### PR TITLE
Chris/detail tests

### PR DESCRIPTION
**Pull request by ChrisAlderson**
Added tests for `episodes` and `torrents` object as described in the documentation of #7.

### CODE REVIEW

File: tests/basic.js [hunk 1]
```diff
@@ -41,12 +41,40 @@ function testDetail(t, d, uniqueId) {
 
     if (type === Provider.ItemType.MOVIE) {
         t.ok(d.trailer, 'we have a trailer');
+
         t.ok(d.torrents, 'we have a torrents field');
+        var quality = Object.keys(d.torrents)[0];
+        t.ok(quality===Provider.QualityType.DEFAULT
+            || quality===Provider.QualityType.LOW
+            || quality===Provider.QualityType.MEDIUM
+            || quality===Provider.QualityType.HIGH,
```
Comment (xaiki) [inline tests/basic.js]: use `quality in Provider.QualityType`

File: tests/basic.js [hunk 2]
```diff
@@ -41,12 +41,40 @@ function testDetail(t, d, uniqueId) {
...
+        t.ok(d.episodes[0].episode, 'we have an episode number');
+        t.ok(d.episodes[0].season, 'we have a season number');
+        t.ok(d.episodes[0].tvdb_id, 'we have a tvdb id');
+
+        t.ok(d.episodes[0].torrents, 'we have a torrents field');
+        var quality = Object.keys(d.episodes[0].torrents)[0];
+        t.ok(quality===Provider.QualityType.DEFAULT
+            || quality===Provider.QualityType.LOW
+            || quality===Provider.QualityType.MEDIUM
+            || quality===Provider.QualityType.HIGH,
```
Comment (xaiki) [inline tests/basic.js]: same

File: tests/basic.js (hunk 1 above)
Comment (ChrisAlderson) [inline tests/basic.js]: What do you mean by this?

Comment (xaiki) [inline tests/basic.js]: you can use a `in` operator, so that when we change the qualities we don't have to rewrite the tests.

**Pull request merged**